pkg/gardenlet/controller/shoot/shoot/reconciler_reconcile.go

```diff
@@ -339,38 +353,44 @@ func (r *Reconciler) runReconcileShootFlow(ctx context.Context, o *operation.Ope
 		})
 		waitUntilShootNamespacesReady = g.Add(flow.Task{
 			Name:         "Waiting until shoot namespaces have been reconciled",
-			Fn:           flow.TaskFn(botanist.Shoot.Components.SystemComponents.Namespaces.Wait).SkipIf(o.Shoot.HibernationEnabled || skipReadiness),
+			Fn:           flow.TaskFn(botanist.Shoot.Components.SystemComponents.Namespaces.Wait),
```

Comment:
You could remove no longer needed `flow.TaskFn()` here and at similar occurrences